libraries/Microsoft.Bot.Builder.Azure/CosmosDbStorageOptions.cs

```diff
@@ -73,5 +74,13 @@ public class CosmosDbStorageOptions
         /// The set of options passed into <see cref="https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.client.documentclient.createdatabaseifnotexistsasync?view=azure-dotnet"/>CreateDatabaseIfNotExistsAsync.
         /// </value>
         public RequestOptions DatabaseCreationRequestOptions { get; set; } = null;
+
+        /// <summary>
+        /// Gets or sets the <a href="https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.idocumentclient?view=azure-dotnet">IDocumentClient</a> to be used internally.
```

Comment:
A comment needs to be added here that this is Optional. If it's not set, the library will create and use the Document Client. Further, we *recommend* developers not set this, and leave the defaults.

Reply:
We added a different `ctor` to avoid changes. We also added a note about the _recommended_ constructor.